meta-koi/recipes-android/android-init/android-init_1.0.bbappend

```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS:prepend:koi := "${THISDIR}/${PN}:"
+
+SRC_URI:append:koi = " file://nonplat_property_contexts \
+    file://plat_property_contexts"
+
+do_install:append:koi() {
+    install -m 0644 ${WORKDIR}/nonplat* ${D}/
+    install -m 0644 ${WORKDIR}/plat* ${D}/
```

Comment:
I'm curious why these files were needed ?
I see you boot your kernel cmdline contains selinux=0 already so selinux labels shouldn't be needed in principle ?

Reply:
koi seems to boot correctly with no files here - should I just delete the files and the bbappend?

Reply:
Okay fair enough